src/accelerate/test_utils/scripts/test_cli.py

```diff
@@ -1,12 +1,19 @@
 import torch
-
+import intel_extension_for_pytorch as ipex
 
 def main():
     if torch.cuda.is_available():
         num_gpus = torch.cuda.device_count()
     else:
         num_gpus = 0
     print(f"Successfully ran on {num_gpus} GPUs")
+    
+    if torch.xpu.is_available():
+        num_xpus = torch.xpu.device_count()
+    else:
+        num_xpus = 0
+    print(f"Successfully ran on {num_xpus} XPUs")
```

Comment:
Should be removed as well, probably.

Reply:
The test scripts will be  cleaned. (includes test code changes inside test_utils)